connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/webhook/InboundWebhookRestController.java

```diff
@@ -139,12 +131,58 @@ protected ResponseEntity verify(
       response =
           new ResponseEntity<>(verificationResult.body(), headers, verificationResult.statusCode());
     }
+
+    return response;
+  }
+
+  private ResponseEntity<?> buildResponse(
```

Comment:
This method seems to have too many branches. Consider maybe splitting it up? 🤔

Reply:
Good point. Pushed fix.